metrotranscom/doorway
# fix: allow the ability to turn on feature in docker

# Pull Request Template

## Issue Overview

This PR addresses #257 


## Description

In order to turn on the feature flag in the build step we need to have it as an argument in docker file

## How Can This Be Tested/Reviewed?

This can't really be tested locally without setting up docker.
*Note*: The feature has already been flipped to on in dev and staging, this change should theoretically pick up that change

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
